manifests/overlays/prod/applications/data_hub/dh-prod-argo.yaml

```diff
@@ -37,3 +37,23 @@ spec:
       selfHeal: true
     syncOptions:
     - Validate=false
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: obslytics.dh-prod-argo
+spec:
+  destination:
+    namespace: dh-prod-thanos-extractor
```

Comment:
Is this namespace part of the cluster spec for the cluster? You can find it under https://github.com/AICoE/aicoe-cd/tree/main/manifests/overlays/prod/secrets

Reply:
Looks like it already is. https://github.com/AICoE/aicoe-cd/commit/e8ebfe83a5483d7734f607d2d79d147474b15f6d

Reply:
Mmmm, I though it should be dh-prod-argo, since it's an application of Fargo and should be executed in its namespace, doesn't it?

Reply:
Ok, i see now the error, fixed